core/CMakeLists.txt

```
@@ -188,6 +195,13 @@ if (WITH_GLFW)
     target_compile_definitions(cubos-core PRIVATE WITH_GLFW)
 endif ()
 
+if (WITH_OPENAL)
+    add_subdirectory(lib/openal-soft)
```

OpenAL has some [settings on by default](https://github.com/kcat/openal-soft/blob/master/CMakeLists.txt#L91-L94) that results in extra tools not needed. These should be disabled. I've only found these so far but there might be more stuff that we do not want:
```suggestion
    set(ALSOFT_UTILS OFF CACHE BOOL "" FORCE)
    set(ALSOFT_NO_CONFIG_UTIL OFF CACHE BOOL "" FORCE)
    set(ALSOFT_EXAMPLES OFF CACHE BOOL "" FORCE)
    add_subdirectory(lib/openal-soft)
```